source/draw/gpu/GpuSurface.ooc

```diff
@@ -81,10 +81,10 @@ GpuSurface: abstract class {
 	_bind: virtual func
 	_unbind: virtual func
 	clear: abstract func
+	blend: func (factor: Float) { this _context blend(factor) }
```

Comment:
OK, I just copied the code from the C# version of Kean where it is a function. Also by doing it this way I didn't have to adapt anything in Vidproc and the old blend hack for monochromes still works. So I thought it was a pretty clean solution for now but I can fix everything in a larger PR if you want me to.